cfpb/sbl-frontend
# feat(update-financial-profile): Error Header

closes #242 

## Changes
- feat(update-financial-profile): Adds Error header upon validation errors

## How to Test
- Navigate to `/update-financial-profile`
- Click on 'Submit'
- Verify that the Error Header is scrolled to
- Verify click on the error link focuses on the specific `InputEntry` with the validation error

## Screenshot
<img width="547" alt="Screenshot 2024-02-08 at 4 10 47 PM" src="https://github.com/cfpb/sbl-frontend/assets/13324863/2c42ab5f-c104-4597-a467-6fbec70cd734">
